### PR TITLE
fix: Fixed the issue that the access_token expired and could not be refreshed

### PR DESCRIPTION
**Pull request by Fleurxxx**
English | [简体中文](https://github.com/opentiny/tiny-engine-webservice/blob/main/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine-webservice/blob/main/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Summary by CodeRabbit

- **New Features**
  - Enhanced AI chat functionality by adding support for `accessToken`, allowing for more personalized and secure interactions.
- **Documentation**
  - Updated documentation comments for the AI chat methods to improve clarity and understanding.

### CODE REVIEW

**Review by lu-yg — APPROVED** [review]

**Comment by CatsAndMice** [comment]
@Lu17301156525 这个PR需要维护者审批才能合并过去

**Review by CatsAndMice — APPROVED** [review]

File: app/service/app-center/aiChat.ts [hunk 1]
```diff
@@ -45,13 +45,13 @@ export default class AiChat extends Service {
     });
   }
 
-  async requestAnswerFromAi(messages: Array<AiMessage>, chatConfig: any) {
+  async requestAnswerFromAi(messages: Array<AiMessage>, chatConfig: any,accessToken: string) {
```
Comment (hexqi) [inline app/service/app-center/aiChat.ts]: 注意代码格式化，参数之间逗号后需要有空格

File: config/config.default.ts [hunk 2]
```diff
@@ -279,7 +279,7 @@ export default (appInfo) => {
         manufacturer: '!openai',
       },
       [E_FOUNDATION_MODEL.ERNIE_BOT_TURBO]: {
-        httpRequestUrl: `https://aip.baidubce.com/rpc/2.0/ai_custom/v1/wenxinworkshop/chat/eb-instant?access_token=${process.env.WENXIN_ACCESS_TOKEN}`,
+        httpRequestUrl: `https://aip.baidubce.com/rpc/2.0/ai_custom/v1/wenxinworkshop/chat/eb-instant?access_token=`+accessToken,
```
Comment (hexqi) [inline config/config.default.ts]: 建议改成：accessToken优先级： 接口传入的accessToken > 环境变量的accessToken，另外建议使用模板字符串替代+拼接字符串
`access_token=${accessToken || process.env.WENXIN_ACCESS_TOKEN}`

File: app/controller/app-center/aiChat.ts [hunk 3]
```diff
@@ -13,14 +13,21 @@ import { Controller } from 'egg';
...
     const { ctx } = this;
-    const { foundationModel, messages } = ctx.request.body;
+    const { foundationModel, messages, accessToken } = ctx.request.body;
     this.ctx.logger.info('ai接口请求参参数 model选型:', foundationModel);
     if (!messages || !Array.isArray(messages)) {
       return this.ctx.helper.getResponseData('Not passing the correct message parameter');
     }
     const model = foundationModel?.model ?? E_FOUNDATION_MODEL.GPT_35_TURBO;
-    ctx.body = await ctx.service.appCenter.aiChat.getAnswerFromAi(messages, { model });
+    ctx.body = await ctx.service.appCenter.aiChat.getAnswerFromAi(messages, { model }, accessToken);
```
Comment (hexqi) [inline app/controller/app-center/aiChat.ts]: accessToken建议整合到第二个参数(ChatConfig)中

**Review by hexqi — APPROVED** [review]

**Comment by CatsAndMice** [comment]
@Lu17301156525 麻烦合并下这个PR

**Pull request merged**